.github/workflows/compliance.yaml

```diff
@@ -0,0 +1,58 @@
+name: Compliance
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+      - unassigned
+
+jobs:
+  check-for-linked-issue:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Check for keyword and issue number
+        id: check-for-keyword
+        uses: actions/github-script@v7
+        with:
+          script: |
+            // Retrieve body of context.payload and search for GitHub keywords and issue number
+            const prBody = context.payload.pull_request.body;
+            const prNumber = context.payload.pull_request.number;
+            const prOwner = context.payload.pull_request.user.login;
+            const regex = /(?<!Example:\s*)(close|closes|closed|fix|fixes|fixed|resolve|resolves|resolved)\s+#(\d+)/i;
```

Comment:
This is really good to make sure we cover all cases, even though our template uses just the "Resolves".

Reply:
Thanks!